lib/velum/kubernetes.rb

```diff
@@ -24,7 +24,7 @@ def self.kubeconfig
                                           arg:         "cat /etc/pki/minion.key"
       host = Minion.master.applied.pluck(:hostname).first
       # rubocop:disable Style/RescueModifier
-      ca_crt = ca["return"].first["ca"]["ca"]["/etc/pki/ca.crt"] rescue nil
+      ca_crt = ca_crt["return"].first.values.first rescue nil
```

Comment:
I would prefer if we used a different variable for this one. If some other code flow is introduced between the initial assignment (line 14 above) and this one, we might end up running only the first assignment and we end up with an attr_accessor which returns an unexpected value. This is and edge case of course but better safe than sorry :).